src/PatientX/run.py

```diff
@@ -26,6 +26,16 @@
 
 app = typer.Typer()
 
+# Chat mode indicates that documents will be sent for each topic one by one. The LLM used for interpretation will
+# maintain context across calls. When using "generate", all documents from all topics will be sent at once and the LLM
+# will provide a topic description for all topics in one go. There may be slight differences in topic quality based on
+# the two options (larger datasets with long documents will likely prefer to use chat)
+#
+# Comment out/uncomment the appropriate mode for your use case
+
+MODE = "generate"
```

Comment:
Nit: This might work better as an enum...  
  
```
from enum import Enum, auto

class Mode(Enum):

    Generate = auto()

    Chat = auto()

mode = Mode.Generate

<later>

if mode == Mode.Generate:
```